readme.md

```diff
@@ -36,7 +36,7 @@ Username to fetch repos from.
 
 #### options
 
-##### token
+##### `token`
```

Comment:
Oh, didn't notice in your previous PR. Can you remove the ticks around `endpoint` instead?